lib/datasets/quora-duplicate-question-pair.rb

```diff
@@ -0,0 +1,56 @@
+require "csv"
+
+require_relative "dataset"
+
+module Datasets
+  class QuoraDuplicateQuestionPair < Dataset
+    Record = Struct.new(:id,
+                        :first_question_id,
+                        :second_question_id,
+                        :first_question,
+                        :second_question,
+                        :duplicated)
+
+    def initialize
+      super()
+      @metadata.id = "quora_duplicate_question_pair"
+      @metadata.name = "quora_duplicate_question_pair"
+      @metadata.url = "https://quoradata.quora.com/First-Quora-Dataset-Release-Question-Pairs"
+      @metadata.licenses = ["Quora Terms of Service"]
+    end
+
+    def each
+      return to_enum(__method__) unless block_given?
+
+      open_data do |csv|
+        csv.each do |row|
+          next if row[0].nil?
+          id,
+          first_question_id,
+          second_question_id,
+          first_question,
+          second_question,
+          duplicated = row.values_at
+
+          record = Record.new(id,
+                              first_question_id,
+                              second_question_id,
+                              first_question,
+                              second_question,
+                              !duplicated.zero?)
```

Comment:
Can we simplify this?

```suggestion
          row["is_duplicate"] = (row["is_duplicate"] == 1)
          record = Record.new(*row.fields)
```

Reply:
fix: [6f1eb21](https://github.com/red-data-tools/red-datasets/pull/149/commits/6f1eb219e5068cd257ebb087aeda4622b8c62601)